### PR TITLE
Task 3.1 reader should be able to view list of books

### PR DESCRIPTION
**Pull request by dmitryProg**

### CODE REVIEW

**Comment by TimWinnicheck** [comment]
Build fails?

File: src/main/java/com/epam/wl/dao/UserOrderDAO.java [hunk 1]
```diff
@@ -39,42 +39,58 @@
...
+//     "SELECT user_order.id AS user_order_id, \"user\".id AS user_id, \"user\".name AS user_name," +
+//             "\"user\".lastname AS user_lastname, \"user\".email AS user_email," +
+//             "book.title AS book_title, book.author AS book_author, book.year AS book_year, user_order.status AS status" +
+//             "FROM user_order INNER JOIN \"user\" ON \"user\".id=user_order.userId INNER JOIN book ON book.id=user_order.bookId WHERE \"user\".id = ?;";
+
+//
+//            "SELECT user_order.id, \"user\".id, \"user\".name, \"user\".lastname, \"user\".email," +
+//            "book.title, book.author, book.year, user_order.status FROM user_order " +
+//            "INNER JOIN \"user\" ON \"user\".id=user_order.userId INNER JOIN book ON book.id=user_order.bookId WHERE \"user\".id = ?";
+
```
Comment (TimWinnicheck) [inline src/main/java/com/epam/wl/dao/UserOrderDAO.java]: Leaving commented out code is a bad practice

File: src/main/java/com/epam/wl/services/TestUserService.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        return users;
+    }
+
+    public void editUser(String name, String lastName, String email, String passwordHash) {
+        Optional<User> user = Optional.of(users.get(0));
+        user.get().setName(name);
+        user.get().setLastname(lastName);
+        user.get().setEmail(email);
+        user.get().setPasswordHash(passwordHash);
+        user.get().setRole(UserRole.USER);//users.get(0).getRole()
```
Comment (TimWinnicheck) [inline src/main/java/com/epam/wl/services/TestUserService.java]: See above

File: src/main/java/com/epam/wl/dao/book_order_handlers/BookOrderListHandler.java [hunk 3]
```diff
@@ -10,17 +11,41 @@
...
+            int bookId = resultSet.getInt("bookid");
+            String author = resultSet.getString("author");
+            String title = resultSet.getString("title");
+            int year = resultSet.getInt("year");
+            int userOrderId = resultSet.getInt("user_orderid");
+            int userId = resultSet.getInt("userid");
+            UserOrderStatus status = UserOrderStatus.valueOf(resultSet.getString("status"));
+            String name = resultSet.getString("name");
+            String lastName = resultSet.getString("lastname");
+            String email = resultSet.getString("email");
```
Comment (ginsenghillock) [inline src/main/java/com/epam/wl/dao/book_order_handlers/BookOrderListHandler.java]: Don't forget finals

**Review by ginsenghillock — COMMENTED** [review]
Main issue - lack of finals

**Comment by TimWinnicheck** [comment]
Reviewed.

**Pull request merged**